src/Centreon/Domain/PlatformInformation/UseCase/V20/UpdatePartiallyPlatformInformation.php

```diff
@@ -121,27 +120,27 @@ public function execute(array $request): void
          * and validate the length of each value.
          * Then transform them as Dto to easily create the PlatformInformation.
```

Comment:
Then transform them as Dto to easily create -> Then transform them as Dto to easily update

Reply:
as we no more use this transition Dto, i'll deleted the line.